rdr_service/model/consent_file.py

```diff
@@ -0,0 +1,42 @@
+from protorpc import messages
+from sqlalchemy import Boolean, Column, Date, event, ForeignKey, Integer, String
+
+from rdr_service.model.base import Base, model_insert_listener, model_update_listener
+from rdr_service.model.participant import Participant
+from rdr_service.model.utils import Enum, UTCDateTime
+
+
+class ConsentType(messages.Enum):
+    PRIMARY = 1
+    CABOR = 2
+    EHR = 3
+    GROR = 4
+
+
+class ConsentSyncStatus(messages.Enum):
+    NEEDS_CORRECTING = 1
+    READY_FOR_SYNC = 2
+
+
+class ConsentFile(Base):
+    __tablename__ = 'consent_file'
+    id = Column("id", Integer, primary_key=True, autoincrement=True, nullable=False)
+    created = Column(UTCDateTime)
+    modified = Column(UTCDateTime)
+    participant_id = Column(Integer, ForeignKey(Participant.participantId))
+    type = Column(Enum(ConsentType))
+
+    file_exists = Column(Boolean)
```

Comment:
Should `file_exists` have a default value of `False`?

Reply:
At the moment I wasn't planning on it. I was planning on having the code explicitly set whether the file was there or not. Do you think it should have a default value?

Reply:
I think it's OK without the default value. I noticed that the other boolean fields had a default value of `False` and I just wanted to make sure this was on purpose.

Reply:
Ah, ok. I see what you mean now, thanks. Yeah, the others default to False to make it easier when creating a validation record for a file that isn't there. That way it can be set to `file_exsts=False` and not need to worry about what should be said for the signature and signing date.